docs/en/How-to-release.md

```diff
@@ -246,3 +244,69 @@ Voting will start now (xxxx date) and will remain open for at least 72 hours, Re
 [ ] +0 No opinion.
 [ ] -1 Do not release this package because....
 ```
+
+## Vote result mail
+Close the vote, if
+1. In 72 hours, you got the more than 3 (+1 binding), and no -1. Vote pass.
+1. Some reviewers found some serious mistakes in this release, the team decided to stop vote and prepare a new RC.
+
+Send a mail to `general@incubator.apache.org` about vote result and status.
+```
+Mail title: [Result][VOTE] Release Apache SkyWalking (incubating) version x.y.z
+
+Mail content:
+Hi all,
+
+The vote for releasing Apache SkyWalking x.y.z (incubating) is closed, now.
+
+Vote result:
+x (+1 binding) (Names of voters)
+y -1.
+
+Thank you everyone for taking the time to review the release and help us. 
+
+I will procedd to publish the release and send ANNOUNCE.
+```
+
+## Publish release
+1. Move source codes tar balls and distributions to `https://dist.apache.org/repos/dist/release/skywalking/`
```

Comment:
maybe an example of how to do the move, eg
`svn mv ^/dev/incubator/skywalking/5.0.0-alpha ^/release/skywalking/`